src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1037,6 +1037,74 @@
         az acr token update -n MyToken -r MyRegistry --scope-map MyNewScopeMap
 """
 
+helps['acr agentpool'] = """
+type: group
+short-summary: Manage private Tasks agent pools with Azure Container Registries.
+"""
+
+helps['acr agentpool create'] = """
+type: command
+short-summary: Create an agent pool for an Azure Container Registry.
+examples:
+  - name: Create the agent pool 'MyAgentName' associated with the registry 'MyRegistry'.
+    text: >
+        az acr agentpool create -n MyAgentName -r MyRegistry
+  - name: Create the agent pool 'MyAgentName' with n agent count.
+    text: >
+        az acr agentpool create -n MyAgentName -r MyRegistry --count n
+  - name: Create the agent pool 'MyAgentName' associated with the registry 'MyRegistry' in VNET 'MyVnetId'.
+    text: >
+        az acr agentpool create -n MyAgentName -r MyRegistry --vnet-id MyVnetId
+"""
+
+helps['acr agentpool update'] = """
+type: command
+short-summary: Update an agent pool for an Azure Container Registry.
+examples:
+  - name: Update the agent pool 'MyAgentName' count
+    text: >
+        az acr agentpool update -n MyAgentName -r MyRegistry --tag NewCount
+  - name: Update the agent pool 'MyAgentName' tag
+    text: >
+        az acr agentpool update -n MyAgentName -r MyRegistry --tag MyTag
```

Comment:
The update is to update the count of agentpool. e.g. az acr agentpool update -n MyAgentName -r MyrRegistry --count 3

Reply:
the two parameters that the SDK accepts are count and tag. But like I said I'll be hiding all the code related to tag for now